paasta_tools/cli/cmds/start_stop_restart.py

```diff
@@ -145,43 +148,48 @@ def issue_state_change_for_service(service_config, force_bounce, desired_state):
 def paasta_start_or_stop(args, desired_state):
     """Requests a change of state to start or stop given branches of a service."""
     instance = args.instance
-    cluster = args.cluster
+    clusters = args.clusters
     soa_dir = args.soa_dir
     service = figure_out_service_name(args=args, soa_dir=soa_dir)
+    if args.clusters is not None:
+        clusters = args.clusters.split(",")
+    else:
+        clusters = list_clusters(service)
+
+    for cluster in clusters:
+        service_config = get_instance_config(
+            service=service,
+            cluster=cluster,
+            instance=instance,
+            soa_dir=soa_dir,
+            load_deployments=False,
+        )
 
-    service_config = get_instance_config(
-        service=service,
-        cluster=cluster,
-        instance=instance,
-        soa_dir=soa_dir,
-        load_deployments=False,
-    )
-
-    try:
-        remote_refs = remote_git.list_remote_refs(utils.get_git_url(service))
-    except remote_git.LSRemoteException as e:
-        msg = (
-            "Error talking to the git server: %s\n"
-            "This PaaSTA command requires access to the git server to operate.\n"
-            "The git server may be down or not reachable from here.\n"
-            "Try again from somewhere where the git server can be reached, "
-            "like your developer environment."
-        ) % str(e)
-        print msg
-        return 1
-
-    if 'refs/heads/paasta-%s' % service_config.get_deploy_group() not in remote_refs:
-        print "No branches found for %s in %s." % \
-            (service_config.get_deploy_group(), remote_refs)
-        print "Has it been deployed there yet?"
-        return 1
-
-    force_bounce = utils.format_timestamp(datetime.datetime.utcnow())
-    issue_state_change_for_service(
-        service_config=service_config,
-        force_bounce=force_bounce,
-        desired_state=desired_state,
-    )
+        try:
```

Comment:
These refs do not change based on the cluster we are operating on. Move this out of the loop. This will also prevent the user from seeing multiple identical error messages in the event that they can't communicate with the git server.